packages/devtools_app/test/test_infra/scenes/performance/default.dart

```diff
@@ -44,10 +44,6 @@ class PerformanceDefaultScene extends Scene {
 
   @override
   String get title => '$PerformanceDefaultScene';
-
-  void tearDown() {
-    FeatureFlags.widgetRebuildstats = false;
-  }
```

Comment:
please leave this. This is a bug that we are not calling tearDown on the scene from tests that use this stager scene

Reply:
Should I add TODO?

Reply:
Done